test/unittests/iotools/test_converter_utils.py

```diff
@@ -0,0 +1,14 @@
+def test_sort_session_list():
+    """Test function `sort_session_list`."""
+
+    from clinica.iotools.converter_utils import sort_session_list
+
+    assert sort_session_list(
+        ["ses-M000", "ses-M006", "ses-M012", "ses-M024", "ses-M048", "ses-M003"]
+    ) == ["ses-M000", "ses-M003", "ses-M006", "ses-M012", "ses-M024", "ses-M048"]
+    assert sort_session_list(
+        ["ses-M00", "ses-M06", "ses-M12", "ses-M24", "ses-M48", "ses-M03"]
+    ) == ["ses-M00", "ses-M03", "ses-M06", "ses-M12", "ses-M24", "ses-M48"]
+    assert sort_session_list(
+        ["ses-M0", "ses-M6", "ses-M12", "ses-M24", "ses-M48", "ses-M3"]
+    ) == ["ses-M0", "ses-M3", "ses-M6", "ses-M12", "ses-M24", "ses-M48"]
```

Comment:
Thanks for adding this !! 👍 
This is a typical example where you might want to use pytest's parametrization:

```python
@pytest.mark.parametrize("input_list,expected",
        [
            (
                ["ses-M000", "ses-M006", "ses-M012", "ses-M024", "ses-M048", "ses-M003"],
                ["ses-M000", "ses-M003", "ses-M006", "ses-M012", "ses-M024", "ses-M048"],
            ),
           (
               ["ses-M00", "ses-M06", "ses-M12", "ses-M24", "ses-M48", "ses-M03"],
               ["ses-M00", "ses-M03", "ses-M06", "ses-M12", "ses-M24", "ses-M48"],
            ),
           (
               ["ses-M0", "ses-M6", "ses-M12", "ses-M24", "ses-M48", "ses-M3"],
              ["ses-M0", "ses-M3", "ses-M6", "ses-M12", "ses-M24", "ses-M48"],
            ),
    ]
)
def test_sort_session_list(input_list, expected):
     """Test function `sort_session_list`."""
     from clinica.iotools.converter_utils import sort_session_list

    assert sort_session_list(input_list) == expected
```

Reply:
Alright ! I'll do it that way then :)

Reply:
Maybe a dumb question but what's specific to this function versus a regular sort? To me, it looks like we are just testing whether the builtin `sorted` function works as expected at the end of the day.

Reply:
I think the added value is in sorting lists of session IDs with heterogeneous number of digits where `sorted` will work differently:

```python
>>> l = ["ses-M003", "ses-M0", "ses-M02"]
>>> sorted(l)
['ses-M0', 'ses-M003', 'ses-M02']
```